sdk/migrationdiscoverysap/Azure.ResourceManager.MigrationDiscoverySap/tests/Tests/MigrationSapDiscoveryTests.cs

```diff
@@ -146,13 +145,16 @@ public async Task PostImportEntities(ArmClient client, SapDiscoverySiteResource
         var operationStatusObj = JObject.Parse(opStatus.Content.ToString());
         var inputExcelSasUri = operationStatusObj?["properties"]?["discoveryExcelSasUri"].ToString();
 
-        //Upload here
-        using (FileStream stream = File.OpenRead(excelPathToImport))
+        if (SessionEnvironment.Mode != RecordedTestMode.Playback)
```

Comment:
please add some comments explaining why we are adding this if, basically this is a data-plane client which can't simply interact with [HttpMockServer](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/mgmtcommon/TestFramework/Microsoft.Azure.Test.HttpRecorder/HttpMockServer.cs).

Reply:
Done